### PR TITLE
Set the correct namespace for SCC when installed in non default namespace

### PR DESCRIPTION
**Pull request by zvonkok**
The SCC has a users field where one can specify the ServiceAccount from a specific Namespace to be used. When installing NFD in a non-default namespace, the SCC ServiceAccount namespace has to be updated.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -64,8 +64,8 @@ undeploy:
 	for obj in $(DEPLOY_OBJECTS) $(DEPLOY_CRDS) $(DEPLOY_CRS); do \
 		$(TEMPLATE_CMD) $$obj | kubectl delete -f - ;\
 	done	
-	 ## Delete everything for the operator from the cluster
-#	-$(TEMPLATE_CMD) $(DEPLOY_OBJECTS) $(DEPLOY_OPERATOR) $(DEPLOY_CRDS) $(DEPLOY_CRS) | kubectl delete -f -
+
+	kubectl delete scc nfd-worker
```
Comment (ArangoGutierrez) [inline Makefile]: Isn't this going to leave the other `$(DEPLOY_OBJECTS) $(DEPLOY_OPERATOR) $(DEPLOY_CRDS) $(DEPLOY_CRS)` objects hanging / not removed?

Comment (zvonkok) [reply Makefile]: This is a comment that is removed look at the resulting Makefile.

**Pull request merged**